app/assets/stylesheets/modules/_footer.scss

```
@@ -1,5 +1,5 @@
 .footer{
-  height: 230px;
+  height: 350px;
   width: 100vw;
   background-color: #272727;
   color: #ffffff;
```

よく使う色はstylesheets/config/_variable.scssのフォルダを作り変数で定義してしまった方が良いかと思います。
```suggestion
  color: #ffffff;
```